libs/icons/src/index.ts

```diff
@@ -26,11 +26,14 @@ export * from './EditLine';
 export * from './ExchangeFunds';
 export * from './ExchangeLine';
 export * from './Expand';
+export * from './GridFill';
```

Comment:
Add the suffix `Icon` when naming new icons to make sure they don't conflict with other components. (e.g. `GridFill` -> `GridFillIcon`)